incubator/consul/templates/consul-petset.yaml

```diff
@@ -157,6 +157,12 @@ spec:
       - name: gossip-key
         secret:
           secretName: gossip-key
+      readinessProbe:
+        httpGet:
+          path: /v1/health/service/consul
```

Comment:
We did a `netstat` to see if Consul was listening to the port, but maybe it is the way we set it up.  I will take it out for a test drive when I have a chance.  But if it is working for you, we can iterate on it in incubator.